std/src/main/scala/algebra/std/map.scala

```diff
@@ -26,33 +50,80 @@ class MapEq[K, V](implicit ev: Eq[V], a: AdditiveMonoid[V]) extends Eq[Map[K, V]
     check(x, y) && check(y, x)
```

Comment:
Yes of course. :blush:

I think maybe I was implementing an ordering and then decided to go with eq but forgot to do this.